Jenkinsfile

```diff
@@ -141,7 +141,7 @@ def startBraveBrowserBuild() {
                                 credentials('brave-builds-github-token-for-pr-builder')
                                 github('brave/devops', 'https')
                             }
-                            branch('master')
+                            branch('gn_check')
```

Comment:
?

Reply:
This is required so the updated code in devops can be used for the CI run: https://github.com/brave/devops/pull/3348 - I'll revert once CI looks all good